include/libdnf/base/goal_elements.hpp

```diff
@@ -77,6 +77,18 @@ enum class GoalProblem : uint32_t {
     ALREADY_INSLALLED = (1 << 13)
 };
 
+enum class GoalAction {
```

Comment:
Unfinished move `Goal::Action` ->`GoalAction`. The original still exists and is used.